terraform/infrastructure/01-variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "name" {
+  type    = string
+  default = "collimator"
+}
+
+variable "environment" {
+  type    = string
+  default = "dev"
+}
+
+variable "region" {
+  type    = string
+  default = "eu-central-2"
+}
+
+variable "domain_name" {
+  type    = string
+  default = "collimator.tyratox.ch"
+}
+
+variable "vpc_cidr" {
+  type    = string
+  default = "10.0.0.0/16"
+}
+
+variable "tags" {
+  type = map(string)
+  default = {
+    Name        = "collimator"
+    Environment = "dev"
+  }
```

Comment:
Why not embed this kind of defaults in the `globals` file?
E.g.
```
locals {
  tags = merge(var.tags, {{
    Application = var.name
    Environment = var.environment
  })
}
```
This also ensures that the user can't override the application/environment tags.